ql/src/java/org/apache/hadoop/hive/ql/exec/tez/TezSessionPoolManager.java

```diff
@@ -119,7 +119,7 @@ public void startPool(HiveConf conf, final WMFullResourcePlan resourcePlan) thro
     if (resourcePlan != null) {
       Collection<String> appliedTriggers = updateTriggers(resourcePlan);
       LOG.info("Updated tez session pool manager with triggers {} from active resource plan: {}",
-          appliedTriggers, resourcePlan.getPlan().getName());
+          appliedTriggers, resourcePlan.getPlan() == null ? "null" : resourcePlan.getPlan().getName());
```

Comment:
Are we sure `resourcePlan` can't be `null`?
Looking at the Hiveserver2 code, it does check for `resourcePlan` as well
https://github.com/apache/hive/blob/98c0a1a047e9c32b8418eee5c53d9413c0ac428d/service/src/java/org/apache/hive/service/server/HiveServer2.java#L933

and 
https://github.com/apache/hive/blob/98c0a1a047e9c32b8418eee5c53d9413c0ac428d/service/src/java/org/apache/hive/service/server/HiveServer2.java#L952